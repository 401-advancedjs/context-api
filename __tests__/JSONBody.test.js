import React from 'react';
import JSONBody from '../component/JSONBody/JSONBody.js';
import renderer from 'react-test-renderer';

describe('<JSONBody />', () => {
  test('basic rendering', () => {
    const mountedJSONBody = shallow(<JSONBody />);
    expect(mountedJSONBody.find('JSONBody')).toBeTruthy();
  });

  test('rendering follows the snapshot', () => {
    const snapshot = renderer.create(<JSONBody />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});