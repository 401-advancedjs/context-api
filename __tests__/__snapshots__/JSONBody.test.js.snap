// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<JSONBody /> rendering follows the snapshot 1`] = `
<textarea
  placeholder="Raw JSON Body"
/>
`;
